modules/ROOT/pages/enabling-einstein.adoc

```diff
@@ -0,0 +1,113 @@
+= Enabling Einstein for Anypoint Platform
+
+When you establish a tenant relationship with a Salesforce organization that has Einstein enabled, you can enable Einstein for your Anypoint Platform organization and store your Einstein generative AI audit and feedback data in Salesforce Data Cloud. You can then use that data for compliance, analytics, prompt adjustments, and LLM tuning to enhance your generative AI applications.
+
+As an organization administrator, you can assign permission to the users in your organization so they can use Einstein with Anypoint Platform products that have Einstein capabilities, including:
+
+* Anypoint Code Builder
+* Intelligent Document Processing (IDP)
+
+
+== Before You Begin
+
+Before enabling Einstein in your Anypoint Platform organization, you must:
+ 
+* Have administrative access to an Anypoint Platform organization that isn't a trial organization and isn't expired
+* Have the Organization Administrator permission and role
+* Have administrative access to a Salesforce organization that https://help.salesforce.com/s/articleView?id=sf.generative_ai_enable.htm&type=5[has Einstein enabled]. 
+* xref:access-management::trusted-salesforce-org.adoc[Establish a tenant relationship] between your Anypoint Platform organization and a trusted Salesforce organization. 
+
+[[enable-einstein-salesforce]]
+== Enable Einstein in the Salesforce Organization
+
+When the Salesforce administrator enables Einstein for the connected Anypoint Platform organization, Einstein requests are routed through the connected Salesforce organization. Billing and tracking are done in the Salesforce organization. 
+
+To enable Einstein for the connected Anypoint Platform organization:
+
+. Log in to your Salesforce organization.
+. In Salesforce Setup, go to the *Quick Find* box and enter `MuleSoft`, then select *MuleSoft > Anypoint Platform Setup*.
+. In *Your connected Anypoint Platform Organization*, click *View Connection Info* to open a new dialog.
+. In *Connection to MuleSoft Anypoint Platform*, switch between *Enabled* or *Disabled* in *Enable Einstein for MuleSoft Anypoint Platform*.
++ 
+image::einstein-enabled-in-salesforce.png["Einstein for MuleSoft Anypoint Platform is enabled"]
+. Click *Close*.
++
+After Einstein is enabled in the Salesforce organization, it's displayed in Anypoint Platform as *Enabled in Salesforce* for the connected Salesforce organization.
+
+[[enable-einstein-anypoint]]
+== Enable Einstein for Your Anypoint Platform Organization
+
+When Einstein is enabled in Anypoint Platform, any Anypoint Platform feature running in that organization is able to use Einstein. The Salesforce administrator can't specify which features are enabled for Einstein, for example, IDP can use Einstein but Anypoint Code Builder can't. 
```

Comment:
```suggestion
When Einstein is enabled in Anypoint Platform, any Anypoint Platform feature running in that organization can use Einstein. The Salesforce administrator can't specify which features are enabled for Einstein, for example, IDP can use Einstein but Anypoint Code Builder can't. 
```

Reply:
What does the last sentence mean? It's enabled at the product level, not the feature level?

Reply:
It's enabled at the org level. So if it's enabled in Anypoint Platform, it's enabled for everything that has an Einstein integration.